common/src/jni/main/cpp/conscrypt/native_crypto.cpp

```diff
@@ -3786,17 +3786,18 @@ static jlong NativeCrypto_X509_get_notAfter(JNIEnv* env, jclass, jlong x509Ref)
     return reinterpret_cast<uintptr_t>(notAfter);
 }
 
-static long NativeCrypto_X509_get_version(JNIEnv*, jclass, jlong x509Ref) {
+static int64_t NativeCrypto_X509_get_version(JNIEnv*, jclass, jlong x509Ref) {
```

Comment:
Both of these are incorrect. It should be `jlong`. These functions prototypes must match the Java type signature at the bottom.